components/Templates/includes/functions-view_template.php

```diff
@@ -522,9 +578,13 @@ function frontier_prefilter_template( $code, $template, $pod ) {
 							$field = trim( $matches[2][ $key ] );
 						}
 						if ( false !== strpos( $field, '.' ) ) {
-							$path  = explode( '.', $field );
-							$field = array_pop( $path );
-							$ID    = '{@' . implode( '.', $path ) . '.' . $pod->api->pod_data['field_id'] . '}';
+							if ( 'if' === $command ) {
+								$ID = '{@' . $pod->api->pod_data['field_id'] . '}';
+							} else {
+								$path  = explode( '.', $field );
+								$field = array_pop( $path );
+								$ID    = '{@' . implode( '.', $path ) . '.' . $pod->api->pod_data['field_id'] . '}';
```

Comment:
:no_entry_sign: Variable "$ID" is not in valid snake_case format, try "$i_d"

<sub>hash: e02ca85ff73494c3f179ba3c81da43d6</sub>